standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/ObjectStore.java

```diff
@@ -10567,6 +10587,16 @@ private StoredProcedure convertToStoredProc(String catName, MStoredProc proc) {
             proc.getSource());
   }
 
+  private Package convertToPackage(String catName, MPackage mPkg) {
+    return new Package(
+            catName,
```

Comment:
doesn't we have something like `mPkg.getDatabase().getCatName()`?

Reply:
Wouldn't it execute and extra select? The DB is joined to the PKG and I think it is loaded lazily.

Reply:
fixed